docs/DeveloperGuide.adoc

```diff
@@ -587,21 +605,21 @@ exist, it is removed from the module.
 
 The `group edit` command has been implemented in a similar manner.
 
-==== Implementation of the Student Add and Delete Commands
+==== Implementation of the Student Add, Delete and Edit Commands
 
 The following sequence diagram shows the sequence of commands that take place
 between the logic and model components of the TA-Tracker when the user enters the
-command `student add n/John Doe p/98765432 e/johnd@example.com m/A0181234G`.
+command `student delete m/A0181234G`.
 
-Note: This diagram assumes that there is no student with the matric number `A0181234G`
-pre-existing in the TA-Tracker.
+Note: This diagram assumes that there is a student with the matric number `A0181234G`
+exists in the TA-Tracker.
```

Comment:
Either say 
"This diagram assumes that a student with the matric number xx exists in ..."
or 
"This diagram assumes that there is a student with matric number xx in the TA-Tracker"

The grammar on this one is a little wrong